index.html

```diff
@@ -113,13 +113,13 @@
     //
     //-------------------------------------------------------------------------
 
```

Comment:
enter a comment in order to start a review.  how are they getting emotes in here? 
`inserting code with the <> ?`